qa/L0_infer_variable/test.sh

```diff
@@ -63,6 +63,17 @@ for TARGET in cpu gpu; do
         done
```

Comment:
Isn't line 62 adding another instance_group for onnx?

Reply:
onnx is not included in the for statement above.

Reply:
Can you put  a comment above the for statement calling that out.